src/App.tsx

```diff
@@ -1,16 +1,18 @@
 import React, { Suspense } from 'react';
 import './App.css';
+import { MOCK_DATA } from './lib/components/PostPreview/post-preview-config';
+import PostPreviewCard from './lib/components/PostPreview/PostPreviewCard';
 import { RenderRoutes } from './navigation/Router';
 import ROUTER_CONFIG from './navigation/router-config';
 
-
 import Header from './lib/components/Header/Header';
 
-const App: React.FC  =() => {
+const App: React.FC = () => {
   return (
     <Suspense fallback={<div>Загрузка...</div>}>
       <div className="App">
-        <Header/>
+        <Header />
+        <PostPreviewCard data={MOCK_DATA} />
```

Comment:
pls remove, it's not necessary, will add that a bit latter :)

Reply:
fixed

Reply:
pls just remove `<PostPreviewCard data={MOCK_DATA} />`

Reply:
we don't need this here for now